boa_engine/src/vm/code_block.rs

```diff
@@ -641,6 +697,18 @@ impl ToInternedString for CodeBlock {
         let mut pc = 0;
         let mut count = 0;
         while pc < self.bytecode.len() {
+            for (i, handler) in self.handlers.iter().enumerate().rev() {
```

Comment:
> Do we need to note the handler start and end in the stack trace?

This is very useful while when debugging, so I'm more inclined to keeping it.

```text
----------------------Compiled Output: '<main>'-----------------------
Location  Count   Handler      Opcode                     Operands

000000    0000       none      PushDeclarativeEnvironment 7
000005    0001       none      LoopContinue               
000006    0002       none      PushTrue                   
000007    0003       none      JumpIfFalse                159
000012    0004       none      PushDeclarativeEnvironment 6
000017    0005    <0: 00??     PushDeclarativeEnvironment 4
000022    0006     1: 0000     PushDeclarativeEnvironment 0
000027    0007     1: 0043     PopEnvironment             
000028    0008     1: 0043     PushOne                    
000029    0009     1: 0043     PushFalse                  
000030    0010     1: 0043     Jump                       69
000035    0011     1: 0043     PopEnvironment             
000036    0012     1: 0043     PushZero                   
000037    0013     1: 0043     PushFalse                  
000038    0014    >1: 0043     Jump                       69
000043    0015    <2: 0067     Exception                  
000044    0016     2: 0067     PushDeclarativeEnvironment 2
000049    0017     2: 0067     PutLexicalValue            0000: 'a'
000054    0018     2: 0067     PushDeclarativeEnvironment 1
000059    0019     2: 0067     PopEnvironment             
000060    0020     2: 0067     PopEnvironment             
000061    0021     2: 0067     PushZero                   
000062    0022     2: 0067     PushFalse                  
000063    0023    >2: 0067     Jump                       69
```

we can have some other kind of more dense notation for the handlers, how about having another column for which handler is responsible for an instruction (if it throws) and the address it would go to. The `<` is start of a handler and `>` end of a handler.

What do you think? @nekevss

Reply:
I like the second notation!